yamcs-web/src/main/webapp/src/app/filetransfer/RemoteFileSelector.ts

```diff
@@ -0,0 +1,215 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, EventEmitter, forwardRef, Input, OnChanges, OnDestroy, Output } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MatTableDataSource } from '@angular/material/table';
+import { BehaviorSubject, Subscription } from 'rxjs';
+import { ListFilesResponse } from '../client';
+
+@Component({
+  selector: 'remote-file-selector',
+  templateUrl: './RemoteFileSelector.html',
+  styleUrls: ['./RemoteFileSelector.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => RemoteFileSelector),
+      multi: true
+    }
+  ]
+})
+export class RemoteFileSelector implements ControlValueAccessor, OnChanges, OnDestroy {
+  // TODO: class almost complete duplicate of ObjectSelector
```

Comment:
FYI, think is fine. IMO is good to have buckets/options entirely seperated from filetransfer